GCEAdapter.py

```diff
@@ -50,11 +51,13 @@ def _configure(self):
         self.min_nodes = self.config['min']
         self.EXPAND_CRITERION = self.config['expand_sensitivity']
         self.SHRINK_CRITERION = self.config['shrink_sensitivity']
-        self.use_gpus = self.config.get("use_gpus", False)
+        self.use_gpus = self.config.get("use_gpus", False) 
         self.CLOUDCUBE_URL = os.environ['CLOUDCUBE_URL']
         self.CLOUDCUBE_ACCESS_KEY_ID = os.environ['CLOUDCUBE_ACCESS_KEY_ID']
         self.CLOUDCUBE_SECRET_ACCESS_KEY = os.environ['CLOUDCUBE_SECRET_ACCESS_KEY']
 
+        self.use_gpus = self.use_gpus == 'True' or self.use_gpus == 'true'
```

Comment:
I would use this:

![image](https://user-images.githubusercontent.com/2185159/57648580-b551af00-757a-11e9-9fa9-d5571b9d111f.png)